Cargo.lock

```diff
@@ -215,6 +221,38 @@ dependencies = [
  "serde",
 ]
 
+[[package]]
+name = "bitcoin"
+version = "0.18.2"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "cd5e7935f613ba170459072926f01dc5ddb8aa22382dc4badf44bbb55e2d243d"
+dependencies = [
+ "bitcoin-bech32",
+ "bitcoin_hashes",
+ "byteorder",
+ "hex 0.3.2",
+ "rand 0.3.23",
+ "secp256k1",
+]
```

Comment:
It seems like we're pulling in a few dependencies which should not be required? Maybe we can feature gate the Bitcion/Ethereum APIs in the trezor library?

Reply:
makes sense, done